addon/components/core/page-numbers.js

```diff
@@ -65,48 +65,69 @@ export default Component.extend({
   currentPageToShow: 1,
   nbPagesToShow: 5,
 
-  pageNumbers: computed('currentPageToShow', 'lastPage', function () {
-    let nbPagesWithoutCurrent = this.get('nbPagesToShow') - 1;
+  currentPage: computed('lastPage', 'currentPageToShow', function () {
```

Comment:
why switch the order? might introduce breaking changes.

Reply:
L'ordre n'a aucune importance dans les watchs

Reply:
Well then